run.py

```diff
@@ -570,11 +679,23 @@ def run_replanning_loop(settings, forecast_year):
 
             # 1a. IF START YEAR, WARM START MANDATORY ACTIVITIES
             if year == start_year:
-                warm_start_activities(settings, year, client)
-                mandatory_activities_generated_this_year = True
+                if demand_model != 'polaris':
+                    warm_start_activities(settings, year, client)
+                    mandatory_activities_generated_this_year = True
+                else:
+                    pilates.polaris.travel_model.run_polaris(None, settings, warm_start=True)
```

Comment:
@jaauld14 can you add `demand_model` as an argument to `warm_start_activities()` and then move this conditional logic and the call to `run_polaris()`to that same method? Trying to keep the executable portion of this run.py script as readable as possible. Eventually the function definitions will probably be re-organized to live elsewhere.

Reply:
Done